src/features/components/fields/HourInputField.js

```diff
@@ -0,0 +1,105 @@
+function generateHours() {
```

Comment:
Refactor suggestion:
1. Let to enter the starting point. For example, if we are booking a meeting in the current date, just let the user to reserve since now.
2. This is more readable if we use two nested loops.

Reply:
Let this for next phase. This needs to update modal depending on its values and I need to check exactly how to do it